### DISCUSSION TITLE
(v4.1 :اصدار) Kodi 20.1لـ Arctic Horizon 2 بيلد عربي

### DISCUSSION THREAD

**Discussion by psp2111-ADSLGATE** [psp2111-ADSLGATE/psp2111-ADSLGATE.github.io#7] [General]
# <p align="center"> بسم الله الرحمن الرحيم </p>

# بيلد Arctic Horizon 2 باللغة العربية لاصدار Kodi 20.1:

#  فيديو 👇
#  [![فيديو](https://i.vimeocdn.com/video/1680745311-6413cc932e67823a80a991c037b26cdb1bdc92c60b39ed2d9d3c0c485f3e22ce-d?mw=2200&mh=1238&q=70)](https://player.vimeo.com/video/834052945?h=9607ab6b6e)

### اصدار: v4.1

-  إضافة نسخة أخرى من البيلد باسم LITE للاجهزة الضعيفة
- اضافة اعدادات كاش افتراصية 300mb للنسخة الكاملة و 100mb لنسخة LITE
-  تعطيل خاصية Dirty regions لتسريع تطبيق كودي قدر الامكان
- تعريب إضافة FEN.
- تغيير مفتاح التقييمات OMDb بمفتاح IMDbList للجميع ومفعل مسبقاً
- تحديث جميع الاضافات الى أخر اصدار وربطها بالمستودع الخاص بي
- اصلاح ظهور نافذة الحلقة التالية لاضافة Seren و Umbrella
- اصلاح أيقونة Disney+
- اصلاح قسم HBO Max
- اضافة أقسام جديدة لخدمات البث
- اضافة قسم جديد باسم (مدير Trakt)
### اصدار: v4.0

- إعادة ترتيب للواجهة بالكامل واضافة أقسام جديدة مثل "خدمات البث" و اخر اصدارات Remux 1080p و 4K
- دعم إضافة Accounts Manager لتفعيل حساب Trakt و Real Debrid لجميع الاضافات بطريقة سهلة وعملية
- إضافة خط جديد لسكن Arctic Horizon 2 مفعل بشكل افتراضي
- تعريب إضافة Accounts Manager
- تقليل جودة البوسترات والخلفيات لتسريع تحميل القوائم وتقليل حجم استهلاك الذاكرة 
- تحديث إضافة الـYouTube لاصلاح مشكلة عدم توفر العرض الدعائي او Trailer لكل الافلام والمسلسلات

## الأخطاء او المشاكل المعروفة:

- خاصية البحث لا تعمل عند الكتابة باللغة العربية (تم اصلاحها النسخ الليلية لكودي 21)
- ...

## متطلبات التشغيل:

- جهاز بمعالج قوي مثل الشيلد أو أقوى
- رام 3 جيجا أو أكثر
- حساب [Real Debrid](http://real-debrid.com/?id=2815547) مدفوع ومفعل
- حساب في موقع [Trakt](https://trakt.tv/) مجاني

# طريقة التثبيت:

روابط تفعيل الحسابات والمستودع: ( راح ترجع لها أثناء الشرح )
رابط المستودع الخاص بي: `https://psp2111-adslgate.github.io/`
Trakt : https://trakt.tv/activate
RealDebrid: https://real-debrid.com/device

## -أولا: تثبيت الويزرد
![1-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/df951587-fe0b-4dc6-b9ec-e44256572ac1)
<img width="640" alt="2-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/5588215d-6f08-4cf9-8533-fc7d258abb2f">
<img width="640" alt="3-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/6c05a8d1-e90e-4ef2-bc68-31ecd76191a1">
<img width="640" alt="4-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/3b024d86-a8a6-4430-b430-d3bae88f70fb">
<img width="640" alt="5-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/4f5b9593-9bbf-4b45-ab0d-47843a2edb9e">
![6-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/68550fb9-6daf-41c8-9df2-693c55cab597)
![7-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/dc0da241-c0d2-4e60-950d-33993f4b82dd)
![8-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/b59aa05f-3cdc-4b92-9c82-03ac5ae8412a)
![9-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/b1d480fc-5144-415f-a27f-9c151edc793a)
![10-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/313ee3e3-cad8-4432-9d16-fbf6dac11ca9)
![11-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/56ff24f0-09ff-4839-a852-ab231d18e608)
![12-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/c6bbeb93-f04d-4fc6-95de-3962e4c5cb17)
![13-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/e6bca1cc-ff67-4d78-9de2-fc5a07ad9f2a)
<img width="640" alt="14-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/6d82c0e0-74b2-465f-9640-560de21bb347">
<img width="640" alt="15-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/d18f668b-6793-4b78-bc2a-a96bd5812706">
<img width="640" alt="16-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/f3452f6c-f216-4c8b-b683-56852fd9fad6">
<img width="640" alt="17-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/fc5f6e3d-63aa-4681-9a49-af4856eaf40a">
<img width="640" alt="18-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/167efc10-e8d7-4793-8cba-a4f22b154f2d">
<img width="640" alt="19-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/7992244c-3f31-4a95-8a73-e5ffcb3ce04c">
<img width="640" alt="20-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/2aedb9ed-953d-4419-b8c2-d89ab13e317f">
<img width="640" alt="21-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/61916735-f18c-455a-913b-c64ef74e8298">
<img width="640" alt="22-min" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/2c906268-d7ec-4ab3-b867-8dc0372412e1">
![23-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/dc0f51d2-afba-4af3-880e-35b9d450288a)

## - ثانياً: تثبيت البيلد من الويزرد
![1-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/6c800c37-43f3-49de-96c3-71e2d8e28eb0)
![2-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/4a80f3ee-5d22-4ac7-a430-4ccea1e46378)
<img width="730" alt="Screenshot 2023-08-12 220103" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/ea71d18a-abde-4879-8665-5db308b792d1">
<img width="735" alt="Screenshot 2023-08-12 220132" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/87a19d62-824c-4657-8bdc-b941686ea561">
<img width="732" alt="Screenshot 2023-08-12 220700" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/ce1c7ac8-5c57-4e39-91c5-467ee4647f5c">
![6-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/58259ec6-4763-46ff-b341-f05340b8f7e2)
![7-min](https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/576e8884-71d0-4c2b-ac86-f4c9ac0068d1)

## - ثالثاً: تفعيل حسابات Trakt و Real Debrid في جميع الاضافات
<img width="640" alt="1" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/7e2e34e0-4ef9-4d8d-a876-198aeb9b00c6">
<img width="640" alt="2" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/f327a84c-c669-41a6-a8e6-4ff8a3579260">
<img width="640" alt="3" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/2ab3498a-5784-4aef-8def-80dc71984163">
<img width="640" alt="4" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/21df1008-a1ee-4670-ace1-314b7a0b8b06">
<img width="576" alt="Screenshot 2023-08-12 230510" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/09fb00dc-a96e-4119-989b-a3e376a50e59">
<img width="561" alt="Screenshot 2023-08-12 230524" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/0923f9be-93a0-4115-be7f-b47b42c32cde">
<img width="640" alt="7" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/35f6b192-ec80-421f-98cd-cfc6f6764092">
<img width="640" alt="8" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/ea0a62ac-71f3-496d-bc33-652e2d22e46a">
<img width="640" alt="9" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/9c70b2d9-5bfd-4593-a761-d60757a2bbe8">
<img width="562" alt="Screenshot 2023-08-12 230250" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/d2357d68-6aed-4037-afad-6cd6298d914d">
<img width="568" alt="Screenshot 2023-08-12 230312" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/4aa07656-210e-4a51-b561-cb1981069216">
<img width="502" alt="Screenshot 2023-08-12 230337" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/3492022a-74f4-4ef2-9c60-7019f009c101">
<img width="640" alt="13" src="https://github.com/psp2111-ADSLGATE/psp2111-ADSLGATE.github.io/assets/80945174/1d3ad05b-2961-4e07-9639-d62fc629d42f">

**Comment by xifaisalix** [comment]


**Reply by psp2111-ADSLGATE** [reply]
لازم تحدث الى Kodi 20 أو أعلى 
تقدر تحمله من موقع Kodi الرسمي [هنا](https://kodi.tv/download/)

**Reply by Abdelkader-bouyd1** [reply]
أولا شكرا جزيلا على هذا العمل الكبير و المميز جداااااا و
ثانيا كل التقدير و الاحترام لهذا الفريق و استمراره في دعم المحتوى العربي الذي يخص برنامج كودي,
و ثالثا نتمنى أن يقوم الفريق بإنشاء مستودعات و اضافات عربية لبرنامج كودي ,
و رابعا نتمنى أن تقوموا بتعريب builds أخرى,
و خامسا ممكن النسخة اللايت الخفيفة من هذا البيلد المعرب و شكرا لكم

**Comment by goodnbad65** [comment]
هل يعمل المستودع مع كودي الجديد 21؟

**Reply by psp2111-ADSLGATE** [reply]
نعم تم تحديثه لدعم Kodi 21

**Reply by Abdelkader-bouyd1** [reply]
أولا شكرا جزيلا على هذا العمل الكبير و المميز جداااااا و
ثانيا كل التقدير و الاحترام لهذا الفريق و استمراره في دعم المحتوى العربي الذي يخص برنامج كودي,
و ثالثا نتمنى أن يقوم الفريق بإنشاء مستودعات و اضافات عربية لبرنامج كودي ,
و رابعا نتمنى أن تقوموا بتعريب builds أخرى,
و خامسا ممكن النسخة اللايت الخفيفة من هذا البيلد المعرب و شكرا لكم

**Comment by PY-MAD** [comment]
جدًا جميل التصميم وأبدعت الله يسعدك لكن هل يمكن اكون في نفس الواجهه ولكن اللغة تكون انجليزية ؟

**Comment by Abdelkader-bouyd1** [comment]
أولا شكرا جزيلا على هذا العمل الكبير و المميز جداااااا و 
ثانيا كل التقدير و الاحترام لهذا الفريق و استمراره في دعم المحتوى العربي الذي يخص برنامج كودي, 
و ثالثا نتمنى أن يقوم الفريق بإنشاء مستودعات و اضافات عربية لبرنامج كودي , 
و رابعا نتمنى أن تقوموا بتعريب builds أخرى, 
و خامسا ممكن النسخة اللايت الخفيفة من هذا البيلد المعرب و شكرا لكم

**Comment by Abdelkader-bouyd1** [comment]
هل هناك جديد يخص برنامج كودي قريبا ستقومون بنشره؟ و شكرا لكم

**Comment by Abdelkader-bouyd1** [comment]
شكرا لكم

**Comment by Youssefhamde39** [comment]
السلام عليكم ورحمة الله اخوكم اول مره اجرب برنامج ده علي الاندرويد ، الحمد لله اتبعت الخطوات كلها تمام والواجهة دي شغالة معايا لكن عندي مشكلة في فتح الفيلم او المسلسل بسبب REAL-DEBRID

بيطلب كود بشغله تمام لكن بعديها في خطوة فيها خانتين اسم مستخدم وكلمة السر ودول مش فاهم بتوع ايه فالتطبيق واقف علي حتي دي ومش عارف اتفرج رجاء المساعدة
ده يوزر تيلجرام BOSS_X7

**Comment by Waleed5711** [comment]
طيب مايمديه يشتغل البرنامج بدون اشتراك trakt

**Reply by psp2111-ADSLGATE** [reply]
ما تحتاج اشترك مدفوع في Trakt 
الحساب العادي او المجاني كافي

فقط تحتاج اشتراك مدفوع في Real Debrid

**Reply by Waleed5711** [reply]
يب ضبط معي يعطيك العافيه طيب تعرف اضافة قنوات مباشرة زي بي ان وطقتها

**Comment by mraba7** [comment]
عندي مشكلة في trakt
كل مرا لازم اعيد البليد من جديد عشان تظهر الافلام و المسلسلات 
![IMG_20250927_200429_550](https://github.com/user-attachments/assets/87826860-70ee-4870-bf09-24c2e9b35d60)

**Comment by l0ykha** [comment]
عندي مشكله ادخل على المسلسل واختار الجوده وانتظر يدجعني للحلقات جربت في كذا مشغل وسويت كلير للكاش مدري وش السالفه

**Comment by Waleed5711** [comment]
يشيخ الحين انا مفعل all-debrid ماشتغل المسسلات مباشر لازم تحملها ليه؟
